### ISSUE TITLE
Solución al problema con la caja de ahorro 

### ISSUE THREAD

**Issue by Tatudelg** [ISPC2020/tscdprog2020_g5-tscdprog2020_g5#3]
El problema que estábamos teniendo cuando llamamos a la caja de ahorro, es que estábamos instanciandola como una clase: 

> cliente.CajaAhorro()

Pero lo que pasa es que ya instanciamos la clase CajaAhorro desde la clase Clientes, en el atributo con el mismo nombre:

> self.CajaAhorro = CajaAhorro(nombre, cantidad)

Es decir, CajaAhorro se instancia sola con el constructor de la clase Clientes y "pasa a ser" un atributo. Por lo que, si queremos ver CajaAhorro hay que llamarla como un atributo, sin los paréntesis finales:

> cliente.CajaAhorro

Para llamar al método imprimir de la clase CajaAhorro:

> cliente.CajaAhorro.imprimir()

**Comment by SilCen** [comment]
Claro, con el tema de CajaAhorro, al poner cliente.CajaAhorro(), estábamos llamando a la clase como si fuera un método de cliente
